src/styles/fonts.sass

```diff
@@ -0,0 +1,13 @@
+@import url('https://fonts.googleapis.com/css?family=Open+Sans:400,600,700|Montserrat:100,200,300,400,500,600,700,800,900')
```

Comment:
Do we really need all these weights?

Reply:
I put all to get more flexibility with the font weights. I will, later on, check and remove the unnecessary weights!